src/AbstractArray.php

```diff
@@ -368,6 +368,18 @@ public function createClone()
     }
 
     /**
+     * Return slice of an array except given keys.
+     * 
+     * @param array $keys List of keys to exclude
+     * 
+     * @return AbstractArray The created array
+     */
+    public function except(array $keys)
+    {
+        return new static(array_diff_key($this->elements, array_fill_keys($keys, 1)));
```

Comment:
You are right - the `array_flip($keys)` will be more appropriate. I use it in `only()` method, you use it all over the Arrayzy code, so there is no reason to use `array_fill_keys($keys, 1)` ;)